app/Console/Commands/VideoData.php

```diff
@@ -98,41 +97,47 @@ public function handle()
 			if (file_exists($url)) {
 				$info = $this->metadataExtractor->extract($url, $photo->type);
 
-				if ($info['width'] !== 0) {
-					$this->line('Extracted metadata');
+				$updated = false;
+				if ($photo->width == 0 && $info['width'] !== 0) {
 					$photo->width = $info['width'];
+					$updated = true;
 				}
-				if ($info['height'] !== 0) {
+				if ($photo->height == 0 && $info['height'] !== 0) {
 					$photo->height = $info['height'];
+					$updated = true;
 				}
-				if ($info['focal'] !== '') {
+				if ($photo->focal == '' && $info['focal'] !== '') {
 					$photo->focal = $info['focal'];
+					$updated = true;
 				}
-				if ($info['aperture'] !== '') {
+				if ($photo->aperture == '' && $info['aperture'] !== '') {
 					$photo->aperture = $info['aperture'];
+					$updated = true;
 				}
-				if ($info['takestamp'] !== null) {
-					$photo->takestamp = $info['takestamp'];
```

Comment:
Presumably this is handled elsewhere so was redundant?

Reply:
We have a separate `lychee:takedate` command specifically for that. Removing it from here makes it more consistent with `lychee:exif_lens` which doesn't do it.